### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by sammou00** [HYF-Class19/agile-development-loruki-group-3#35] [General]
Review

**Comment by sammou00** [comment]
Hello, well done.
some notes : 

1- Note sure what do you mean by `Ui design` , `HTML`and `CSS` in external constrains , but they shouldn't be there 
![image](https://user-images.githubusercontent.com/67711948/182946965-0760c5a8-82fd-419f-92ea-9ceab974d9cb.png)

2-  very good `backlog` and `strategy-plan`

3- very good `design` 

4- very good `user-personas` 

5- nothing in your `index.html` and `index.css` 
![image](https://user-images.githubusercontent.com/67711948/182947563-5a465ae1-0170-4e85-82be-3a82807780c4.png)

![image](https://user-images.githubusercontent.com/67711948/182947640-aa81795a-9a0c-487d-b10e-19b41470f498.png)

**Comment by anasabbar** [comment]
we just started working in the html and css . wish us the best !!

**Comment by johnedelbi** [comment]
@samirm00 you can check our work inside the **(pages)** folder in the root folder 


for UI design we mean that we have already the template and we will rebuild the code on that UI (Loruki template).

**Comment by sammou00** [comment]
Hello, Well done.

1- You can add General styles at the top of your css file 
![image](https://user-images.githubusercontent.com/67711948/183111243-5c44034a-a9a8-4c5a-9796-b6f97c95385e.png)

2-  No need for navbar start and end , just add navbar 
![image](https://user-images.githubusercontent.com/67711948/183111427-23d7fc16-e131-4bea-9a33-c7ba1ca3ea27.png)

The rest is great.
